Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -399,6 +399,8 @@ std::optional<FontPlatformData::Attributes> ArgumentCoder<FontPlatformData::Attr
 #endif
 
 #if ENABLE(WIRELESS_PLAYBACK_TARGET)
+#if !PLATFORM(COCOA)
```

Comment:
WIRELESS_PLAYBACK_TARGET is only enabled on Cocoa ports so I don't understand the point of these `!PLATFORM(COCOA)` checks.

Reply:
Fixed